.github/workflows/test.yaml

```diff
@@ -0,0 +1,79 @@
+name: Testing
+
+on: [push, pull_request]
+
+jobs:
+  unit-tests:
```

Comment:
https://github.com/juju/python-libjuju/pull/774#discussion_r1034096482 might be added here in this PR instead too